benchmarking/chain-sync/configuration/log-config-byron-proxy.yaml

```diff
@@ -18,6 +18,9 @@ hasEKG: 12789
 
 # if wanted, the Prometheus interface is using this port:
 hasPrometheus: 12799
+#hasPrometheus: 
+#  - "localhost"
+#  - 12799
```

Comment:
I don't like seeing things comment out.

Reply:
This is in preparation. The byron-proxy has not updated its dependencies yet.